build/manifest-templates/clusterserviceversion.yaml

```diff
@@ -14,6 +14,7 @@ metadata:
     containerImage: __IMAGE_SRC__
     createdAt: __DATE__
     support: Red Hat, Inc. 
+    olm.skipRange: ">=1.0.2 <__VERSION__"
     alm-examples: |-
       [
         {
```

Comment:
There are some missing commas on that JSON.
Lines: 23, 68 and 82

Reply:
fixed.